ru/turing/courses/lesson2/animals/Bird.java

```diff
@@ -0,0 +1,19 @@
+package ru.turing.courses.lesson2.animals;
+
+public class Bird extends Animal {
+    private boolean migration;
+    //true - letit na south,false-no
+    private String singing;
+
+    public Bird(String vid, boolean eatingmeat, int age, boolean wild, boolean migration, String singing) {
```

Comment:
```suggestion
    public Bird(String type, boolean eatingmeat, int age, boolean wild, boolean migration, String singing) {
```
Когда переименовываешь переменные, надо это делать через рефакторинг, либо вручную потом менять названия в оставшихся местах